apps/app/components/NotificationSwitch/BaseSwitch.tsx

```diff
@@ -5,6 +5,7 @@ import { forwardRef } from 'react'
 export interface SwitchProps extends FlexProps {
   checked?: boolean
   handleStyle?: MotionStyle
+  inactiveBg?: string
```

Comment:
You should use `bg` directly. Because `bg` in `FlexProps` can cover original `bg`.